playwright.config.js

```diff
@@ -1,6 +1,6 @@
 const config = {
   webServer: {
-    command: "npm run start",
+    command: "npm run start -- --single",
```

Comment:
what is this command for?

Reply:
This puts sirv into "single page application" mode, which we need for URLs like "/apps/fenix/pings/metrics" to work. I'll add a comment

https://github.com/lukeed/sirv/tree/master/packages/sirv-cli